bundles/org.openhab.binding.unifi/README.md

```diff
@@ -239,9 +239,17 @@ The default mode value is `auto`.
 
 The `accessPoint` information that is retrieved is available as these channels:
 
-| Channel ID | Item Type | Description                        | Permissions |
-|------------|-----------|------------------------------------|-------------|
-| enable     | Switch    | Enable or disable the access point | Read, Write |
+| Channel ID | Item Type            | Description                                                          | Permissions |
+|------------|----------------------|----------------------------------------------------------------------|-------------|
+| online     | Switch               | Online status of the device                                          | Read        |
+| enable     | Switch               | Enable or disable the access point                                   | Read, Write |
+| name       | String               | Name of device (from the controller web UI)                          | Read        |
+| site       | String               | Site name (from the controller web UI) the device is associated with | Read        |
+| macAddress | String               | MAC address of the device                                            | Read        |
```

Comment:
I'm wondering if MAC address is relevant, since the Thing is configured using that MAC address? It cannot change, because the Thing would then no longer be online.

Reply:
Yeah I agree. It should only change when you change the configuration of the thing. I guess it can go then 🤷‍♂️

Reply:
Yes, it's useless when it can never change.

Reply:
I removed the channel (everywhere I hope)